ValetTests/ValetTests.swift

```diff
@@ -0,0 +1,742 @@
+//
+//  ValetTests.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/25/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import XCTest
+
+
+extension NSError
+{
+    // The NSError.code -> VALMigrationError conversion is gross right now:
+    var valetMigrationError: VALMigrationError { return VALMigrationError(rawValue: UInt(self.code))! }
```

Comment:
Kinda gross, since this line could crash if `valetMigrationError` is used incorrectly. But... I'm okay with `!`s in test code.